db/schema.rb

```diff
@@ -50,7 +51,6 @@
     t.integer "defence_statement"
     t.integer "number_of_witnesses"
     t.string "supplemental_claim"
-    t.string "preparation_time"
```

Comment:
why was this removed? I would assume that it will break something?

Reply:
added back